src/backend/api/downloadmanager.ts

```diff
@@ -84,8 +84,8 @@ export const handleDMQueueInformation = (
 export const cancelDownload = (removeDownloaded: boolean) =>
   ipcRenderer.send('cancelDownload', removeDownloaded)
 
-export const cancelExtraction = (appName: string) =>
-  ipcRenderer.send('cancelExtraction', appName)
+export const cancelExtraction = (appName: string, removeDownloaded: boolean) =>
```

Comment:
I think we should pass an object instead of a flag.

```ts
canceExtraction = ({ appName, removeDownloaded }: {appName: string; removeDownloaded: boolean }) => {...}
```